app/lib/registries/base_registries.rb

```diff
@@ -5,7 +5,10 @@ class BaseRegistries
     def all
       @all ||= {
         'world_locations' => world_locations,
-        'part_of_taxonomy_tree' => topic_taxonomy
+        'all_part_of_taxonomy_tree' => topic_taxonomy,
+        'part_of_taxonomy_tree' => topic_taxonomy,
+        'people' => people,
+        'organisations' => organisations,
```

Comment:
I wasn't sure whether adding things here would cause them to be used to populate the facets in finders.  This might be fine, but might also not be ideal yet.

Reply:
After this change we will still default to the title value that rummager provides us with for people, organisations and world locations facets.

```
    label = value.fetch("title", find_facet_title_by_slug(slug, facet_key))
```